docs/creating-service-bindings/creating-service-binding.md

```diff
@@ -5,7 +5,204 @@ sidebar_position: 1
 # Creating Service Binding
 
 In this section, we consider that the service you want to connect to is already exposing binding metadata by either:
-- Beeing part of the "known" services
+- Being part of the "known" services
 - Exposing binding-metadatas by default
 If it's not the case, please refer to the following section(#)
 
+
+### Service Binding Resource
+
+Service Binding resources use `binding.operators.coreos.com` group with  `v1alpha1` version and `ServiceBinding` kind.
+
+A sample Service Binding resource that connects a nodejs application to a postgresql database:
+
+
+```yaml
+apiVersion: binding.operators.coreos.com/v1alpha1
+kind: ServiceBinding
+
+metadata:
+  name: binding-request 
+
+spec:
+
+  application:
+    name: nodejs-rest-http-crud
+    group: apps
+    version: v1
+    resource: deployments
+
+  services:
+  - group: postgresql.dev
+    version: v1alpha1
+    kind: Database
+    name: db-demo
+```
+
+The `spec` of a Service Binding resource has two sections:
+- application
+- services
+
+### Application
+
+An application is a process running within a container. Examples include a NodeJS Express application, a Ruby Rails application or a Spring Boot application.
```

Comment:
I like that you are giving an example deployment config. 
But here, we should explain the "application" section from the ServiceBinding CR. We should detail each possible fields and what they are representing + explanation of the format expected. Like definition of an API.

Reply:
The reason that I didn't add field explaination over here is because I thought this is the place where field descriptions are supposed to reside -> https://github.com/slemeur/service-binding-documentation/blob/main/docs/creating-service-bindings/service-binding-resource.md

However, I have added the field description here.

Let me know what else could I add here.